example/interface.js

```diff
@@ -348,6 +349,20 @@ import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer, Uuid } f
         return avatarMixer.myAvatar.position;
     };
 
+    // Entity Server
```

Comment:
Move this section up to above "// Wire up mixers."

Reply:
Fixed.